### ISSUE TITLE
Adminer access needs Adminer not Root (Joomla TKL 15.5 anyway)

### ISSUE THREAD

**Issue by l-arnold** [turnkeylinux/tracker#1370]
This may be by design but it was not obvious to me having just set the MYSQL password (twice in fact) and still not being able to get into Adminer.

I logged into Webmin, looked at MYSQL (ala mysql) Users and saw Adminer there.  That worked for login to Adminer with the mysql password.

I have not seen it documented, nor does it show in ConfConsole or Turnkey-Init scripts.

Probably should be noted in ConfConsole if that is the approach going forward.

**Comment by JedMeister** [comment]
Yes it was a change in v15.0. FWIW, that's because by default the root user connection is via a socket. That means that no password is required when working on the commandline (as root). But it also means it's not possible to use the root MySQL/MariaDB user, if not logged in as the root user (e.g. Adminer - which incidentally runs as user 'www-data').

It is noted on all the appliance pages that include Adminer, e.g. on the [LAMP](https://www.turnkeylinux.org/lamp) page. And the first boot script (i.e. inithooks) should also explicitly be asking for you to set the 'adminer' user password?!

Although perhaps it could be documented further/elsewhere? I'm open to suggestions. Having said that, I expect that you missed it because you're an experienced TurnKey user used to using 'root'. :smile:

**Comment by OnGle** [comment]
Non bug with resolution. Closing.
